inst/emails/contact_initial_ans

```diff
@@ -2,15 +2,19 @@ Dear %s,
+The NSF-funded Arctic Data Center support team is passionate about Arctic science. We are here to support researchers like yourself in storing and sharing valuable data.
 
 We are contacting you today because projects funded by the NSF Arctic Sciences Section have specific requirements for data and metadata storage. The Arctic Data Center is a long-lived, publicly accessible archive that was created to help researchers meet the NSF data and metadata storage requirements.
 
-At this time, please note that NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are often required to be submitted on that same timeframe (although the timing and submission requirements for data varies between programs). More detailed information about what will be expected can be found at https://arcticdata.io/submit/#who-must-submit.
+NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are often required to be submitted on that same timeframe. More detailed information about what will be expected can be found at: https://arcticdata.io/submit/#who-must-submit.
 
-With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Typically, NSF researchers already do a great job of collecting data and metadata, but you will want to ensure your data and metadata are collected in a manner that can be easily accessed and understood by other researchers after it is published. We recommend that you visit our data submission guidelines page (https://arcticdata.io/submit/) to get a better understanding of what is expected for submissions to the Arctic Data Center.
+With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Some guidelines are provided here: https://arcticdata.io/submit/#organizing-your-data. If you are interested, we also have annual training opportunities posted here: https://arcticdata.io/training/.
 
-If you have any questions about the Arctic Data Center or NSF requirements, please reply to this email and we will respond as soon as possible. We will also contact you again near the end of your award with reminders about the data submission requirements.
+If your research will involve large data submissions (a total size of 500MB or more), please get in touch with us so we can create a plan together for those datasets.
```

Comment:
I think this should be a higher limit. Maybe > 100GB total size, or more than 500 files. They should be able to handle a few hundred GB uploads through the web form with ease. What was the reason for making the limit so low?

Reply:
Changed! I think this was a typo from a message I was referring to